schematic/models/validate_attribute.py

```diff
@@ -1186,10 +1185,13 @@ def url_validation(
                     try:
                         # Check that the URL points to a working webpage
                         # if not log the appropriate error.
-                        request = Request(url)
-                        response = urlopen(request)
                         valid_url = True
-                        response_code = response.getcode()
+                        response = requests.options(url, allow_redirects=True)
```

Comment:
For the FAIR team, was there a specific reason to use `urllib` other than the fact that it is a builtin package?

I _think_ something like this works too
```
from urllib import request

req = request.Request(url, method='OPTIONS')
response = request.urlopen(req)
print(response.status, response.headers)
```

Reply:
I see. I also prefer using requests package. What Brian has there seems perfect to me.